### PR TITLE
[Merged by Bors] - Removed tools image

### PR DESCRIPTION
**Pull request by maltesander**
# Description

Removed tools image. Working locally with locally build docker image.

## Review Checklist

- [ ] Code contains useful comments
- [ ] CRD change approved (or not applicable)
- [ ] (Integration-)Test cases added (or not applicable)
- [ ] Documentation added (or not applicable)
- [ ] Changelog updated (or not applicable)
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
- [ ] Helm chart can be installed and deployed operator works (or not applicable)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ All notable changes to this project will be documented in this file.
 - Upgrade to `operator-rs` `0.27.1` ([#347])
 - Do not run init container as root anymore and avoid chmod and chown ([#353])
 - [BREAKING] Use Product image selection instead of version. `spec.version` has been replaced by `spec.image` ([#356])
+- [BREAKING]: Removed tools image from init container and replaced with Druid product image. This means the latest stackable version has to be used in the product image selection ([#358])
```
Comment (sbernauer) [inline CHANGELOG.md:20]:
```suggestion
- [BREAKING] Removed tools image from init container and replaced with Druid product image. This means the latest stackable version has to be used in the product image selection ([#358])
```

**Review by sbernauer — APPROVED** [review]
lgtm when tests pass

**Comment by maltesander** [comment]
test https://ci.stackable.tech/view/02%20Operator%20Tests%20(custom)/job/druid-operator-it-custom/57/

**Comment by maltesander** [comment]
bors merge

**Pull request closed without merging**